src/DatadogLoggingService.js

```diff
@@ -35,8 +35,8 @@ class DatadogLoggingService extends NewRelicLoggingService {
       service: process.env.DATADOG_SERVICE,
       env: process.env.DATADOG_ENV,
       version: process.env.DATADOG_VERSION,
-      sessionSampleRate: 50,
-      sessionReplaySampleRate: 100,
+      sessionSampleRate: parseInt(process.env.SESSION_SAMPLE_RATE || 0, 10),
```

Comment:
Should the 0 be the default? Should that be 50?

Reply:
do we want `sessionSampleRate` default  to 50 for all MFEs? if yes then i'll update it